user/admin.py

```diff
@@ -1,5 +1,27 @@
 from django.contrib import admin
-from django.contrib.auth.admin import UserAdmin
+from django.contrib.auth.admin import UserAdmin as DjangoUserAdmin
+from django.utils.translation import gettext as _
+
 from .models import User
 
-admin.site.register(User, UserAdmin)
+
+@admin.register(User)
+class UserAdmin(DjangoUserAdmin):
+    """Define admin model for custom User model with no email field."""
```

Comment:
The docstring mentions 'with no email field', but the `email` field is included in both `fieldsets` and `add_fieldsets`. Ensure the docstring accurately reflects the fields present in the `User` model.